prog/types.go

```diff
@@ -181,6 +181,16 @@ type IntTypeCommon struct {
 	BitfieldMdl bool
 }
 
+// Returns the size of integer in native format or 8 for string-formatted integers.The return
+// value is used in computing limits and truncating other values. This can be improved in the
```

Comment:
Move the "This can be improved" part into the function body where we actually do this part that can be improved and prefix with "TODO:".